packaging/change_name.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+"""
+Barebones utility for changing the name of the package in `pyproject.toml`.
+
+Usage
+-----
+[SCRIPT] [PYPROJECT_FILE] [NAME]
```

Comment:
The script can print its own, accurate, help message.

Reply:
Good spot, I was writing this in an ad-hoc fashion and didn't bother changing it once it worked.